js/src/redux/providers/status.js

```diff
@@ -155,24 +155,24 @@ export default class Status {
 
     const { refreshStatus } = this._store.getState().nodeStatus;
 
-    const statusPromises = [ this._api.eth.syncing() ];
+    const statusPromises = [ this._api.eth.syncing(), this._api.parity.netPeers() ];
 
     if (refreshStatus) {
       statusPromises.push(this._api.eth.hashrate());
-      statusPromises.push(this._api.parity.netPeers());
     }
 
     Promise
       .all(statusPromises)
       .then((statusResults) => {
```

Comment:
Agreed here with @derhuerst - put the optional ones at the back, check for undefined if you must (then not adding it), but just cleaner.